### ISSUE TITLE
[Request]  1.16.5 support

### ISSUE THREAD

**Issue by piousminion** [Sil3ntWxlf/carpentersblocksreborn#3]
The title says it all.  Can we get a 1.16.5 build?   1.16.5 is almost certainly the last version of the 1.16.x line and most other mods are using it.

Thanks for keeping this project live.

**Comment by Sil3ntWxlf** [comment]
Certainly! I do plan on updating to 1.16.5 for the only version before 1.17. Anything besides 1.16.5 will be updated as soon as possible. The projects will all be found on my github in due time. Releases will be both on Forge and Fabric as soon as I am able to port them across!
